src/components/views/settings/tabs/user/PreferencesUserSettingsTab.tsx

```diff
@@ -246,6 +263,14 @@ export default class PreferencesUserSettingsTab extends React.Component<IProps,
                 label={_t('Show tray icon and minimise window to it on close')} />;
         }
 
+        let disableHardwareAccelerationOption = null;
+        if (this.state.disableHardwareAccelerationSupported) {
+            disableHardwareAccelerationOption = <LabelledToggleSwitch
+                value={this.state.disableHardwareAcceleration}
+                onChange={this.onDisableHardwareAccelerationChange}
+                label={_t('Disable hardware acceleration (requires restart to take effect)')} />;
```

Comment:
IIRC Nad said that its better to always phrase settings in the positive (and change the default accordingly)

Maybe changing this should auto-restart given some users get confused by Close (to tray) vs Quit/Exit